vault/auth.go

```diff
@@ -893,6 +914,13 @@ func (c *Core) setupCredentials(ctx context.Context) error {
 				}
 			})
 		}
+
+		// Update the last mounted version and persist
+		entry.LastMounted = c.currentVaultVersion.Version
+		if err := c.persistAuth(ctx, c.auth, &entry.Local); err != nil {
+			c.logger.Error("failed to persist last mounted version to auth table", "error", err)
```

Comment:
I thought (probably incorrectly) that if we couldn't persist here we should just continue operating withoutwriting out the mount entries. I think if this is the case, we'll want to error out, as it could result in some post-update drift and later a demand to downgrade.